raiden/network/transport/matrix/__init__.py

```diff
@@ -1,31 +1,25 @@
 import json
-import re
```

Comment:
Kind of a nitpick: Would you mind renaming this to `transport.py` (or similar) and just importing the relevant bits to `__init__`?